### ISSUE TITLE
WINDOWS: Throws error while creating block

### ISSUE THREAD

**Issue by MrSaqibAmeen** [ahmadawais/create-guten-block#6]
### Is this a bug report?
Yes

### Environment

1. `node -v`: 9.2.0
2. `npm -v`: 5.5.1
3. `yarn --version` (if you use Yarn): NA
4. `npm ls cgb-scripts` (if you haven’t ejected): not sure about this one. In plugins directory, it gives `-- (empty)

Then, specify:

1. Operating system: Windows 8.1
2. Browser and version (if relevant): NA
3. Terminal: GitBash

### Steps to Reproduce

1. Open GitBash, type `npm install create-guten-block --global`
2. Go to plugin directory of local WP installation, type `create-guten-block my-block`

### Expected Behavior

It should create plugin `my-block`

### Actual Behavior

Throw error:

![create-guten-block](https://user-images.githubusercontent.com/31374163/35147591-f486bc2e-fd30-11e7-9e8d-6f0cd8a6bfb9.PNG)

```
$ create-guten-block beta-test-block


📦   Creating a WP Gutenberg Block plguin called:  beta-test-block

 In the directory: C:\Users\Saqib Ameen\Documents\Websites\www.wpcouplelabs.dev\                                wp-content\plugins/beta-test-block
 This might take a couple of minutes.

C:\Users\Saqib Ameen\AppData\Roaming\npm\node_modules\create-guten-block\index.j                                s:60
        throw err;
        ^

TypeError: this.stream.clearLine is not a function
    at Ora.clear (C:\Users\Saqib Ameen\AppData\Roaming\npm\node_modules\create-g                                uten-block\node_modules\ora\index.js:53:15)
    at Ora.render (C:\Users\Saqib Ameen\AppData\Roaming\npm\node_modules\create-                                guten-block\node_modules\ora\index.js:59:8)
    at Ora.start (C:\Users\Saqib Ameen\AppData\Roaming\npm\node_modules\create-g                                uten-block\node_modules\ora\index.js:74:8)
    at module.exports (C:\Users\Saqib Ameen\AppData\Roaming\npm\node_modules\cre                                ate-guten-block\app\run.js:41:10)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at Function.Module.runMain (module.js:684:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:613:3
```

### Reproducible Demo

NA

**Comment by MrSaqibAmeen** [comment]
`npx create-guten-block my-block` throws following error:

```
$ npx create-guten-block my-block
Error: EPERM: operation not permitted, mkdir 'C:\Users\Saqib'
TypeError: Cannot read property 'get' of undefined
    at errorHandler (C:\Program Files\nodejs\node_modules\npm\lib\utils\error-handler.js:205:18)
    at C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js:83:20
    at cb (C:\Program Files\nodejs\node_modules\npm\lib\npm.js:214:22)
    at C:\Program Files\nodejs\node_modules\npm\lib\npm.js:252:24
    at C:\Program Files\nodejs\node_modules\npm\lib\config\core.js:81:7
    at Array.forEach (<anonymous>)
    at C:\Program Files\nodejs\node_modules\npm\lib\config\core.js:80:13
    at f (C:\Program Files\nodejs\node_modules\npm\node_modules\once\once.js:25:25)
    at afterExtras (C:\Program Files\nodejs\node_modules\npm\lib\config\core.js:178:20)
    at C:\Program Files\nodejs\node_modules\npm\node_modules\mkdirp\index.js:47:53
C:\Program Files\nodejs\node_modules\npm\lib\utils\error-handler.js:205
  if (npm.config.get('json')) {
                 ^

TypeError: Cannot read property 'get' of undefined
    at process.errorHandler (C:\Program Files\nodejs\node_modules\npm\lib\utils\error-handler.js:205:18         )
    at process.emit (events.js:159:13)
    at process._fatalException (bootstrap_node.js:382:26)
Install for prefix@latest failed with code 7
C:\Users\Saqib Ameen\AppData\Roaming\npm\node_modules\create-guten-block\index.js


📦   Creating a WP Gutenberg Block plguin called:  my-block

 In the directory: C:\Users\Saqib Ameen\Documents\Websites\www.wpcouplelabs.dev\wp-content\plugins/my-b         lock
 This might take a couple of minutes.

this.stream.clearLine is not a function
```

**Comment by ahmadawais** [comment]
@MrSaqibAmeen Can you also add the error you got previously with the command you ran?

**Comment by MrSaqibAmeen** [comment]
I have added below the screenshot.

**Comment by ahmadawais** [comment]
It seems like the issue is with Ora package.

```js
TypeError: this.stream.clearLine is not a function
```
Kindly, take a look.

Which is why on Windows I have to do this

```js

const isWindows = require( 'is-windows' );
const ora = isWindows() ? false : require( 'ora' );

// Init the spinner.
	const spinner = ora ? new ora( { text: '', enabled: true } ) : false;

if ( spinner ) {
		spinner.start(
			`1. Creating the plugin directory called → ${ chalk.black.bgWhite(
				` ${ blockName } `
			) }`
		);
	} else {
		console.log(
			chalk.green( ' 1. Creating the plugin directory called → ' ) +
				chalk.black.bgWhite( ` ${ blockName } ` )
		);
	}

await createPluginDir( blockName );

	if ( spinner ) {
		spinner.succeed();
	}
```

Which as you can see is less than ideal.

**Comment by ahmadawais** [comment]
Thanks for the report @MrSaqibAmeen fixed in 261612a5e29bec4b58fca40f2f3a80b462d798d9
